lib/WorldState.js

```diff
@@ -98,7 +100,7 @@ function parseArray(ParserClass, dataArray, deps, uniqueField) {
 /**
  * Parses Warframe Worldstate JSON
  */
-module.exports = class WorldState {
+class WorldState {
```

Comment:
```suggestion
export default class WorldState {
```

Reply:
This class is already being exported on line 311.